scipy/spatial/tests/test_spherical_voronoi.py

```diff
@@ -183,8 +183,7 @@ def test_rank_deficient(self):
     @pytest.mark.parametrize("radius", [0.5, 1, 2])
     @pytest.mark.parametrize("center", [(0, 0, 0), (1, 2, 3)])
     def test_geodesic_input(self, n, radius, center):
-        np.random.seed(0)
-        U = Rotation.random().as_matrix()
+        U = Rotation.random(random_state=0).as_matrix()
```

Comment:
@nmayorov Can you confirm this is an improvement? I have a hard time keeping track of the changes in `random` machinery in NumPy & what is going on with `Rotation` class.

Reply:
I think this is a usual approach to ensure test repeatability.

Nothing special here, looks reasonable to me.

Reply:
I guess I'm not clear on the difference before vs. after. Anyway, if you aren't worried that's good.